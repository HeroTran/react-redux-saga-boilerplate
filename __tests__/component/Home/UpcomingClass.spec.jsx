/*
import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import UpcomingClass from '../../../app/renderer/components/Home/UpcomingClass';

let wrapper;
beforeEach(() => {
  wrapper = (
    <MemoryRouter>
      <UpcomingClass
        key={'b6991a15-f9ea-4746-af0c-d637dd66ee8e'}
        roomId={'b6991a15-f9ea-4746-af0c-d637dd66ee8e'}
        href={`classroom/${'b6991a15-f9ea-4746-af0c-d637dd66ee8e'}`}
        name={'learnerName'}
        userTime={'2020-05-31 13:00'}
        learnerNextLevel={null}
        bookingType={1}
      />
    </MemoryRouter>
  );
});
describe('should be rendered without error', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
*/
