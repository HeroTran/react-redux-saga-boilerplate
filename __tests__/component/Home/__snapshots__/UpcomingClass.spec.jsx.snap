// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be rendered without error matches snapshot 1`] = `
<div
  className="main-upcoming"
>
  <a
    href="/classroom/b6991a15-f9ea-4746-af0c-d637dd66ee8e"
    onClick={[Function]}
  >
    <div
      className="each-card"
    >
      <div
        className="header-card"
      >
        <i />
        <p>
          Upcoming Class
        </p>
      </div>
      <div
        className="info-card"
      >
        <p>
          <span
            className="info"
          >
            Learner :
          </span>
          <span
            className="detail"
          >
            learnerName
          </span>
        </p>
        <p>
          <span
            className="info"
          >
            Date Time :
          </span>
          <span
            className="detail"
          >
            2020-05-31 13:00
          </span>
        </p>
        <p>
          <span
            className="info"
          >
            Lesson Name :
          </span>
          <span
            className="detail"
          >
            N/A
          </span>
        </p>
        <p
          className="booking-type"
        >
          Trial
        </p>
      </div>
    </div>
  </a>
</div>
`;
